src/plugins/intel_gpu/src/kernel_selector/cl_kernels/permute_f_y_axes.cl

```diff
@@ -77,13 +82,17 @@ KERNEL (permute_f_y_axes)(
     for (int j = 0; j < FEATURE_BLOCK_SIZE; ++j) {
         __attribute__((opencl_unroll_hint(TILE_SIZE)))
         for (int i = 0; i < TILE_SIZE; ++i) {
-            const int x_idx = x_begin + i;
-            const int y_idx = y_begin + j;
+            int x_idx = x_begin + i;
+            int y_idx = y_begin + j;
             const uint input_offset = INPUT0_GET_INDEX(b_idx, f_idx, y_idx, x_idx) - get_sub_group_local_id();
             INPUT0_TYPE res = DT_INPUT_BLOCK_READ(input, input_offset);
     #if HAS_FUSED_OPS
+            f_idx = y_begin + j;
+            y_idx = bf % INPUT0_FEATURE_NUM;
             FUSED_OPS;
             transpose_buf[bf_local][j][i]  = FUSED_OPS_RESULT;
+            f_idx = bf % INPUT0_FEATURE_NUM;
```

Comment:
It shouldn't affect match in case of permute kernel, but I think it would be better to keep original `f_idx` unchanged with const modifier instead of its' recalculation FEATURE_BLOCK_SIZE * TILE_SIZE times, and introduce new variable for fused ops calculation only - you just need to properly configure `original_output_order ` descriptor

Reply:
done for f_idx and y_idx